### PR TITLE
Update supported Ubuntu LTS version and related instructions

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Ditto.

### CODE REVIEW

File: guide/linux.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 It may happen that the main window of Webots is not refreshed properly and
 appears blank at startup or upon resize or maximization. This is caused by a
-conflict between the Compiz window manager and OpenGL. Simply disabling Compiz
-should fix such a problem. This can be achieved on Ubuntu Linux from the System
-menu: `Preferences / Appearance / Visual Effects / None`.
+conflict between the Compiz window manager and OpenGL.
+Simply disabling the operating system's visual effecs should fix such a problem.
```
Comment (omichel) [inline guide/linux.md]: typo: effecs

File: guide/system-requirements.md [hunk 2]
```diff
@@ -19,20 +19,19 @@ Webots on your computer systems to ensure they are compatible before you buy.
 The following operating systems are supported:
 
 - Linux: Webots is ensured to run on the latest Ubuntu Long Term Support (LTS)
-release, currently version 14.04. But it is also known to run on most recent
+release, currently versions 16.04. But it is also known to run on most recent
```
Comment (omichel) [inline guide/system-requirements.md]: why did you add an "s" to version?

Comment (stefaniapedrazzi) [inline guide/system-requirements.md]: Because initially I kept the version 14.04, but then I decided to remove it and I didn't notice that the s remained.
